docs/multi-index-search.md

```diff
@@ -53,13 +54,14 @@ Let's take a look at what a minimalistic example of multi-index search looks lik
 </template>
 ```
 
-In this example, we have 2 indices 2 search boxes and 2 different result sets.
+In this example, we display results from two indices, but we are still using two search boxes.
 
-Changing the text in the first input, will only update the first results.
+## Correlated Multi-Index searches
 
-### Correlated Multi-Index searches
+Here's how to have a single search box displaying results from multiple indices:
 
-Here is a different scenario:
+@TODO: this is not using the search box widget, why? This seems to be a different
```

Comment:
As for now, there is no way to use the search box in this scenario given it has to be wrapped into an index component.
Still need to iterate and find the best way to address this without bloating the API.

One solution is to wrap everything in an index component, and wrap only an other part in another index.